pkg/controller/resourcequota/resource_quota_monitor.go

```diff
@@ -352,7 +352,7 @@ func (qm *QuotaMonitor) processResourceChanges() bool {
 		utilruntime.HandleError(fmt.Errorf("cannot access obj: %v", err))
 		return true
 	}
-	klog.V(4).Infof("QuotaMonitor process object: %s, namespace %s, name %s, uid %s, event type %v", event.gvr.String(), accessor.GetNamespace(), accessor.GetName(), string(accessor.GetUID()), event.eventType)
+	klog.V(4).InfoS("QuotaMonitor processing object", "obj", event.gvr.String(), "namespace", accessor.GetNamespace(), accessor.GetName(), "UID", string(accessor.GetUID()), "eventType", event.eventType)
```

Comment:
I think this is a wrong format?
by the way,you can use klog.KRef.

Reply:
Fixed.